Source/WebCore/animation/AnimationEffectTiming.cpp

```diff
@@ -62,7 +62,7 @@ void AnimationEffectTiming::updateComputedProperties(std::optional<WebAnimationT
             // Set start delay and end delay to 0, as it is not possible to mix time and proportions.
             startDelay = WebAnimationTime::fromPercentage(0);
             endDelay = WebAnimationTime::fromPercentage(0);
-            iterationDuration = std::isinf(iterations) ? WebAnimationTime::fromPercentage(0) : *timelineDuration;
+            iterationDuration = std::isinf(iterations) ? WebAnimationTime::fromPercentage(0) : *timelineDuration / iterations;
```

Comment:
There's no step in https://drafts.csswg.org/web-animations-2/#time-based-animation-to-a-proportional-animation which actually defines this it seems. I'll approve as the logic seems fine and the test passes, but I'm a bit confused from how this maps from the spec.